operator/main.go

```diff
@@ -126,7 +132,6 @@ func run() error {
 	if !enableWebhooks.BooleanSetting() {
```

Comment:
I think it's worth to flip this if clause to make it similar to the new added `if enableWebhooks.BooleanSetting()`. I mean it might be confusing to first check if it's true, then check if it's false and in the else part create something

Reply:
Makes sense, but let me raise another PR for this, where I won't have to do the "wait for CI/resolve go.sum conflicts" loop again.

Reply:
https://github.com/stackrox/stackrox/pull/7848